docs/docs/Introduction/intro.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Introduction to FlutterFlow.
+---
+
+# Introduction
+
+Welcome To FlutterFlow Docs!
+
+## What is FlutterFlow?
+
+FlutterFlow is a low-code builder for developing native mobile applications. You can use our simple drag-and-drop interface to build your app 10x faster than traditional development.
+
+<div class="video-container"><iframe src="https://www.youtube.com/embed/GpXjU-ieAKU?si=moIEUUGry24CdSJN" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></div>
+
+## New to FlutterFlow?
+
+The [Getting Started](/getting-started/dashboard) section will give you a quick overview of the FlutterFlow interface and guide you on building your first app with FlutterFlow.
+
+If there are any topics you'd like us to add to our documentation, please share your feedback [here](https://flutterflow.typeform.com/to/hxg5nxbo).
```

Comment:
Maybe we point to our documentation GitHub repo and allow users to suggest topics there